LEAF_Request_Portal/sources/Group.php

```diff
@@ -325,20 +344,21 @@ public function addMember($member, $groupID): string
      *
      * Created at: 9/15/2022, 9:30:20 AM (America/New_York)
      */
-    public function importUser(string $userID, int $groupID, string|null $backupID): array
+    public function importUser(string $userID, int $groupID, string $backupID): array
     {
         $this->dataActionLogger->logAction(\Leaf\DataActions::ADD, \Leaf\LoggableTypes::EMPLOYEE, [
             new \Leaf\LogItem("users", "userID", $userID, $this->getEmployeeDisplay($userID)),
             new \Leaf\LogItem("users", "groupID", $groupID, $this->getGroupName($groupID))
         ]);
 
-        $sql_vars = array(':userID' => $userID,
+        $vars = array(':userID' => $userID,
                         ':groupID' => $groupID,
                         ':backupID' => $backupID,);
+        $sql = 'INSERT INTO users (groupID, userID, backupID)
+                VALUES (:groupID, :userID, :backupID)
+                ON DUPLICATE KEY UPDATE userID=:userID';
 
-        $result = $this->db->prepared_query('INSERT INTO users (groupID, userID, backupID)
-                                                    VALUES (:groupID, :userID, :backupID)
-                                                    ON DUPLICATE KEY UPDATE userID=:userID', $sql_vars);
+        $result = $this->db->pdo_insert_query($sql, $vars);
 
         return (array) $result;
```

Comment:
Same here?

Reply:
This one already is, the pdo_insert_query returns a properly formatted json response.